lib/seattleflu/db/cli/command/etl/longitudinal.py

```diff
@@ -380,38 +381,31 @@ def symptoms(document: dict) -> list:
     parent_reported_symptoms = document.get("sx_specific") or []
     rn_reported_symptoms = document.get("rn_sx") or []
 
-    symptoms = list(set(parent_reported_symptoms + rn_reported_symptoms))
+    symptoms = parent_reported_symptoms + rn_reported_symptoms
```

Comment:
Using `set()` here to de-duplicate is too early now that we standardize names into multiple possible things below, so I moved it to the end.